### ISSUE TITLE
Doesn't convert arrays properly to the values file, using hardcoded values

### ISSUE THREAD

**Issue by jjvillal** [arttor/helmify#56]
If you got something like this in the kustomize file:

`authorizedUsers: ["user1", "user2"]`

Or like this:

```
authorizedUsers:
  - user1
  - user2
 ```

Then is parsed as the same value directly on the template, like a hardcoded value, instead of creating this array on the values.yaml and using it on the template as every other variable does.

Expected in the template example:

`authorizedUsers: {{ toYaml .Values.authorizedUsers | nindent 8 }}`

**Comment by arttor** [comment]
hi, @jjvillal can you please attach your source.yaml manifest (configmap?) aloing with actual and expected helmify output. it will help to write test and fix issue.

**Comment by nishyt** [comment]
Hi @arttor as requested , please find the source.yaml along with the actual and expected Output

**Source.yaml**
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: test-cm
data:
  test_config.yaml: | 
    serverDetails:
      tempFilePath: /tmp/.ssh/test_file
      filetransferTargets: 
        - scp://guest@1.1.1.1:22/home/defaultuser/.ssh/authorized_keys
        - scp://guest@2.2.2.2:22/home/defaultuser/.ssh/authorized_keys
        - scp://guest@3.3.3.3:22/home/defaultuser/.ssh/authorized_keys
      proxyAddress: 1.1.1.1
      port: 22
      serverUser: guest
      privateKey: "/home/testuser/.ssh/id_rsa"
      publicKey: "/home/testuser/.ssh/id_rsa.pub"
```
**Actual Output [/templates/test-cm.yaml]:** 
NOTE: the field which is passed as an list/array (**here filetransferTargets**), is getting parsed as the same value directly inside the template rather than being getting added in values.yaml file
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: {{ include "cmchart.fullname" . }}-test-cm
  labels:
  {{- include "cmchart.labels" . | nindent 4 }}
data:
  test_config.yaml: |
    serverDetails:
      filetransferTargets:
      - scp://guest@1.1.1.1:22/home/defaultuser/.ssh/authorized_keys
      - scp://guest@2.2.2.2:22/home/defaultuser/.ssh/authorized_keys
      - scp://guest@3.3.3.3:22/home/defaultuser/.ssh/authorized_keys
      port: {{ .Values.testCm.testConfigYaml.serverDetails.port }}
      privateKey: {{ .Values.testCm.testConfigYaml.serverDetails.privateKey | quote }}
      proxyAddress: {{ .Values.testCm.testConfigYaml.serverDetails.proxyAddress | quote
        }}
      publicKey: {{ .Values.testCm.testConfigYaml.serverDetails.publicKey | quote }}
      serverUser: {{ .Values.testCm.testConfigYaml.serverDetails.serverUser | quote }}
      tempFilePath: {{ .Values.testCm.testConfigYaml.serverDetails.tempFilePath | quote
        }}
```
**Actual values.yaml**
```
kubernetesClusterDomain: cluster.local
testCm:
  testConfigYaml:
    serverDetails:
      port: 22
      privateKey: /home/testuser/.ssh/id_rsa
      proxyAddress: 1.1.1.1
      publicKey: /home/testuser/.ssh/id_rsa.pub
      serverUser: guest
      tempFilePath: /tmp/.ssh/test_file
```

---------------------------------------------------------------
**Expected Output**
NOTE:  The expectation is to have the list/array values being getting added to values.yaml file as it is there in the source.yaml
and then is parsed inside the template as shown below.
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: {{ include "cmchart.fullname" . }}-test-cm
  labels:
  {{- include "cmchart.labels" . | nindent 4 }}
data:
  test_config.yaml: |
    serverDetails:
      filetransferTargets: {{ toYaml .Values.testCm.testConfigYaml.ServerDetails.filetransferTargets | nindent 8 }}
      port: {{ .Values.testCm.testConfigYaml.serverDetails.port }}
      privateKey: {{ .Values.testCm.testConfigYaml.serverDetails.privateKey | quote }}
      proxyAddress: {{ .Values.testCm.testConfigYaml.serverDetails.proxyAddress | quote
        }}
      publicKey: {{ .Values.testCm.testConfigYaml.serverDetails.publicKey | quote }}
      serverUser: {{ .Values.testCm.testConfigYaml.serverDetails.serverUser | quote }}
      tempFilePath: {{ .Values.testCm.testConfigYaml.serverDetails.tempFilePath | quote
        }}
``` 
**Expected values.yaml**
```
kubernetesClusterDomain: cluster.local
testCm:
  testConfigYaml:
    serverDetails:
      filetransferTargets:
        - scp://guest@1.1.1.1:22/home/defaultuser/.ssh/authorized_keys
        - scp://guest@2.2.2.2:22/home/defaultuser/.ssh/authorized_keys
        - scp://guest@3.3.3.3:22/home/defaultuser/.ssh/authorized_keys
      port: 22
      privateKey: /home/testuser/.ssh/id_rsa
      proxyAddress: 1.1.1.1
      publicKey: /home/testuser/.ssh/id_rsa.pub
      serverUser: guest
      tempFilePath: /tmp/.ssh/test_file

**Comment by arttor** [comment]
@nishyt thank you for your contribution! Chahges available in [v0.3.21 release](https://github.com/arttor/helmify/releases/tag/v0.3.21).
